### ISSUE TITLE
TypeError: _jss.create is not a function

### ISSUE THREAD

**Issue by zerkalica** [rollup/rollup#2159]
Problem with jss and react-jss.

Rollup from 0.58 generates ``` var _jss = ( lib$1 && index$1 ) || lib$1; ```

Where lib$1 is jss exports
```js
	var lib$1 = /*#__PURE__*/Object.freeze({
		default: index$1,
		__moduleExports: lib,
		create: lib_1,
```

And index$1 - default jss export

```js
var index$1 = unwrapExports(lib);
```

But react react-jss imports non-default create export:

```js
exports['default'] = (0, _jss.create)((0, _jssPresetDefault2['default'])());
```
This causes exception:
```
a.out.js:7806 Uncaught TypeError: (0 , _jss.create) is not a function
    at a.out.js:7806
    at createCommonjsModule (a.out.js:11)
    at a.out.js:7766
    at a.out.js:8694
```

Demo repo: https://github.com/zerkalica/rollup-bug

rollup 0.57.2 works fine.

**Comment by guybedford** [comment]
Are you sure the problem here is with updating Rollup itself and not just the CommonJS plugin? Have you tried just downgrading the CommonJS plugin version?

**Comment by zerkalica** [comment]
Sure. Reproduced with rollup-plugin-commonjs@8.4.1 and 9.1.0

It's regression bug in rollup 0.57.1 -> 0.58.
rollup 0.57.1 - bug not reproduced, only 0.58.2.

**Comment by guybedford** [comment]
Thanks, I finally tracked this down and posted a fix at https://github.com/rollup/rollup-plugin-commonjs/pull/313.

**Comment by zerkalica** [comment]
Fixed in 0.59
